src/lib/containers/UserSearchBoxV2.tsx

```diff
@@ -108,6 +110,13 @@ const UserSearchBoxV2: React.FC<UserSearchBoxProps> = props => {
     typeFilter,
   )
 
+  const selectRef = React.useRef<any>(null)
```

Comment:
Set this as `any` because
- when left empty the below code `selectRef.current.focus()` would have a type `never`.
- when setting it as SelectInstance as the docs showed, the propTypes were incompatible with the `customSelectComponents`.
- vice versa when matching the `customSelectComponents` was incompatible with the RefAttributes.